ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos/ip-hardware.j2

```diff
@@ -0,0 +1,7 @@
+{# eos - IP Hardware #}
+{% if ip_hardware is arista.avd.defined %}
+!
+{%   if ip_hardware.fib.optimize.prefix_profile is arista.avd.defined %}
```

Comment:
indentation error. Should be 4 spaces with respect to outer if. But you gave only 2 spaces. Please check in all templates.